packages/clerk-js/src/utils/captcha/turnstile.ts

```diff
@@ -69,21 +71,38 @@ export const shouldRetryTurnstileErrorCode = (errorCode: string) => {
   return !!codesWithRetries.find(w => errorCode.startsWith(w));
 };
 
-async function loadCaptcha(url: string) {
+async function loadCaptcha(fallbackUrl: string) {
   if (!window.turnstile) {
     try {
-      await loadScript(url, { defer: true });
+      await loadCaptchaFromCloudflareURL();
     } catch {
-      // Rethrow with specific message
-      console.error('Clerk: Failed to load the CAPTCHA script from the URL: ', url);
-      throw {
-        captchaError: 'captcha_script_failed_to_load',
-      };
+      await loadCaptchaFromFAPIProxiedURL(fallbackUrl);
     }
   }
   return window.turnstile;
 }
 
+async function loadCaptchaFromCloudflareURL() {
+  try {
+    await loadScript(CLOUDFLARE_TURNSTILE_ORIGINAL_URL, { defer: true });
+  } catch (err) {
+    console.error('Clerk: Failed to load the CAPTCHA script from the original Cloudflare URL.');
```

Comment:
Let's also log the original error here as well, might be useful for debugging during the rollout phase

Reply:
I was thinking about totally removing this console error, to be honest! Chrome is already throwing an error on the console.